src/advanced/downloader.md

```diff
@@ -3,13 +3,7 @@
 
 The Downloader is a service responsible for seeding and downloading historical data using the BitTorrent protocol. Data is stored in the form of immutable `.seg` files, known as **snapshots**. The Ethereum core instructs the Downloader to download specific files, identified by their unique info hashes, which include both block headers and block bodies. The Downloader then communicates with the BitTorrent network to retrieve the necessary files, as specified by the Ethereum core.
 
-<div class="warning">
-
-**Information**:
-
-While all Erigon components are separable and can be run on different machines, the Downloader must run on the same machine as Erigon to be able to share downloaded and seeded files.
-
-</div>
+> **Warning**: While all Erigon components are separable and can be run on different machines, the Downloader must run on the same machine as Erigon to be able to share downloaded and seeded files.
```

Comment:
```suggestion
> **Warning**: Erigon components can operate independently by communicating over the network except for the Downloader, which must share a filesystem with the Erigon node.
```

Reply:
@AskAlexSharov it's just the filesystem that must be shared right? Technically if that's a network filesystem like nfs, sshfs, or some other fs shared by machines that would work fine?